content/develop/connect/insight/debugging.md

```diff
@@ -0,0 +1,95 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Redis Insight debugging information
+linkTitle: Debugging information
+stack: true
+title: Redis Insight debugging information
+weight: 2
+---
+
+If you are experiencing errors or other issues when using Redis Insight, follow the steps below to learn more about the errors and to identify root cause.
+
+## Connection issues
+
+If you experience connection issues, try these steps.
+
+### I. Launch Redis Insight in debug mode
+
+Run the following command to launch Redis Insight in debug mode to investigate connection issues:
+
+* **Windows**:
+
+    `cmd /C “set DEBUG=ioredis* && ".\Redis Insight.exe"”`
+
+* **macOS** (from the Applications folder):
+
+    `DEBUG=ioredis* open "Redis Insight.app"`
+
+* **Linux**:
+
+    `DEBUG=ioredis* "redis insight"`
+
+### II. Investigate logs
+
+You can review the Redis Insight log files (files with a `.log` extension) to get detailed information about system issues.
+These are the locations on supported platforms:
+
+- **Docker**: In the `/data/logs` directory *inside the container*.
+- **macOS**: In the `/Users/<your-username>/.redis-insight` directory.
+- **Windows**: In the `C:\Users\<your-username>\.redis-insight` directory.
+- **Linux**: In the `/home/<your-username>/.redis-insight` directory.
+
+## Other issues
+### To debug issues other than connectivity:
```

Comment:
This one has a colon while the others don't (or of course, you could add a colon to the others if that works better).
```suggestion
### To debug issues other than connectivity
```